README.rst

```diff
@@ -0,0 +1,83 @@
+Example: Basic Sphinx project for Read the Docs
```

Comment:
Guessing this file meant to be in this PR?

Reply:
Ah sorry no - I was just using it to copy from, as mkdocs is Markdown-only, we can use a README.md.